pyeudiw/tools/utils.py

```diff
@@ -20,8 +19,8 @@ def iat_now() -> int:
 
 
 def exp_from_now(minutes: int = 33) -> int:
-    _now = timezone.localtime()
-    return int((_now + datetime.timedelta(minutes=minutes)).timestamp())
+    now = datetime.datetime.now()
```

Comment:
.now is timezone naive

we need a now datetime timezone aware

Reply:
Resolved in 648d5c0